gulpfile.babel.js

```diff
@@ -0,0 +1,105 @@
+import gulp from 'gulp';
+import util from 'gulp-util';
+import minimist from 'minimist';
+import rename from 'gulp-rename';
+import replace from 'gulp-replace';
+import path from 'path';
+import del from 'del';
+import coffee from 'gulp-coffee';
+import stream from 'vinyl-source-stream';
+import browserify from 'browserify';
+import coffeeify from 'coffeeify';
+import shim from 'browserify-shim';
+import { Server as Karma } from 'karma';
+
+import { version } from './package';
+import { standalone } from './package';
+import { filename } from './package';
+
+
+const source = './src/brainstem.coffee';
+const options = minimist(process.argv.slice(2));
+
+const moduleOutput = './lib';
+const gemOutput = './vendor/assets/javascripts';
+
+
+// Tasks
+
+gulp.task('build-module', () => {
```

Comment:
This is handled via `.gitignore` and `.npmignore` respectively. However, this is a good point for the legacy gem support output.